File: vortexdbg-api/src/main/java/com/vortexdbg/debugger/gdb/EnableExtendedModeCommand.java
package com.vortexdbg.debugger.gdb;

import com.vortexdbg.Emulator;

class EnableExtendedModeCommand implements GdbStubCommand {

    @Override
    public boolean processCommand(Emulator<?> emulator, GdbStub stub, String command) {
        stub.makePacketAndSend("OK");
        return true;
    }

}

File: vortexdbg-api/src/main/kotlin/com/vortexdbg/debugger/gdb/EnableExtendedModeCommand.kt
package com.vortexdbg.debugger.gdb

import com.vortexdbg.Emulator

internal class EnableExtendedModeCommand : GdbStubCommand {

    override fun processCommand(emulator: Emulator<*>, stub: GdbStub, command: String): Boolean {
        stub.makePacketAndSend("OK")
        return true
    }

}
